options/options.js

```diff
@@ -23,70 +29,71 @@ $(() => {
+      let user = item.user,
+        domain, userLink, tokenLink;
 
-    if(item.scm === 'bitbucket') {
+      if (item.scm === 'bitbucket') {
         domain = '@Bitbucket.org';
         userLink = `https://bitbucket.org/${user}`;
         tokenLink = `https://bitbucket.org/account/user/${user}/api`;
-    } else if(item.scm === 'gitlab') {
-      if (item.baseUrl !== 'https://gitlab.com/api/v4') {
-        let match = item.baseUrl.match(/:\/\/(.*)\/api\/v4/);
-        if (!match || !match[1]) {
-          domain = '';
-          userLink = '';
-          tokenLink = '';
+      } else if (item.scm === 'gitlab') {
+        if (item.baseUrl !== 'https://gitlab.com/api/v4') {
+          let match = item.baseUrl.match(/:\/\/(.*)\/api\/v4/);
+          if (!match || !match[1]) {
+            domain = '';
+            userLink = '';
+            tokenLink = '';
+          } else {
+            domain = `@${match[1]}`;
+            userLink = `https://${match[1]}/${user}`;
+            tokenLink = `https://${match[1]}/profile/personal_access_tokens`;
+          }
         } else {
-          domain = `@${match[1]}`;
-          userLink = `https://${match[1]}/${user}`;
-          tokenLink = `https://${match[1]}/profile/personal_access_tokens`;
+          domain = '@gitlab.com';
+          userLink = `https://gitlab.com/${user}`;
+          tokenLink = `https://gitlab.com/profile/personal_access_tokens`;
         }
       } else {
-        domain = '@gitlab.com';
-        userLink = `https://gitlab.com/${user}`;
-        tokenLink = `https://gitlab.com/profile/personal_access_tokens`;
-      }
-    } else {
         domain = '@Github.com';
         userLink = `https://github.com/${item.user}`;
         tokenLink = 'https://github.com/settings/tokens';
         if (item.baseUrl !== 'https://api.github.com') {
-            let match = item.baseUrl.match(/:\/\/(.*)\/api\/v3/);
-            if (!match || !match[1]) {
-                domain = '';
-                userLink = '';
-                tokenLink = '';
-            } else {
-                domain = `@${match[1]}`;
-                userLink = `https://${match[1]}/${item.user}`;
-                tokenLink = `https://${match[1]}/settings/tokens`;
-            }
+          let match = item.baseUrl.match(/:\/\/(.*)\/api\/v3/);
+          if (!match || !match[1]) {
+            domain = '';
+            userLink = '';
+            tokenLink = '';
+          } else {
+            domain = `@${match[1]}`;
+            userLink = `https://${match[1]}/${item.user}`;
+            tokenLink = `https://${match[1]}/settings/tokens`;
+          }
         }
-    }
+      }
 
-    $('#login-user').text(`${user}${domain}`).attr('href', userLink);
-    $('#token').attr('href', tokenLink);
-  })
-  .catch(err => {
-    //not logged in
-  })
+      $('#login-user').text(`${user}${domain}`).attr('href', userLink);
+      $('#token').attr('href', tokenLink);
+    })
+    .catch(err => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/083dcd77ddb6869d2b7dc2cdf2f7452c410d4901/options/options.js#L80"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>